### PR TITLE
Precision improvements

### PR DESCRIPTION
**Pull request by cpennington**
@alawibaba @macdiesel @doctoryes: Review?

### CODE REVIEW

File: locust/stats.py [hunk 1]
```diff
@@ -647,8 +648,9 @@ def print_percentile_stats(stats):
                 stats.percentile_column_name(percentile)
             )
             data.append_col([
+                # We add 1e-9 to value, to prevent a division by zero error
                 "{0} (+{1:.2%}/-{2:.2%})".format(
-                    value, float(upper)/value, float(lower)/value
+                    value, float(upper)/(value + 1e-9), float(lower)/(value + 1e-9)
```
Comment (doctoryes) [inline locust/stats.py]: Somewhat related: Could you round the precision of the confidence intervals returned above in `get_percentile_confidence_intervals` as well?

Also, search for `Configence`.

File: locust/test/test_stats.py [hunk 2]
```diff
@@ -117,6 +117,8 @@ def test_serialize_through_message(self):
         self.assertEqual(20, u1.median_response_time)
 
     def test_default_response_time_rounding(self):
+        self.assertEquals(self.s._round_response_time(0.1234), 0.1)
```
Comment (doctoryes) [inline locust/test/test_stats.py]: If the default precision is 2, why isn't the correct answer `0.12` here?

Comment (cpennington) [inline locust/test/test_stats.py]: Because I'm fudging it? For some reason, you get one less actual digit when it's a value less than 1.

**Comment by doctoryes** [comment]
:+1:

**Comment by macdiesel** [comment]
:+1:

**Pull request merged**